template/kubeflow/notebooks/profile-controller_access-management/kustomization.yaml

```diff
@@ -6,6 +6,7 @@ resources:
 
 patchesStrategicMerge:
```

Comment:
Not a comment  against anything, just moderate surprise that it uses the one in kubeflow/kubeflow. But it _was_ done by DavidSpek and after looking into the manifest myself it does seem like they are one in the same